srcpkgs/tcc/template

```diff
@@ -11,12 +11,12 @@ make_check_target="test"
 hostmakedepends="perl"
 short_desc="Tiny C Compiler"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="LGPL-2.1"
+license="LGPL-2.1-only"
 homepage="http://bellard.org/tcc/"
 distfiles="http://repo.or.cz/tinycc.git/snapshot/${_gitrev}.tar.gz>${pkgname}-${version}.tar.gz"
 checksum=24b34f5ff46694cc3da7db5df6f6180e8cba908e6e3691577f01dd17f4f1a6b4
 nopie=yes
-nocross=yes
+nostrip=yes
```

Comment:
Always provide a reasoning when adding `nostrip` And I have the feeling that it is wrong in thise case.